src/sql/media/icons/common-icons.css

```diff
@@ -308,23 +308,22 @@
 	background-image: url('stop_cell_solidanimation_inverse.svg');
 }
 
-.codicon.arrow-up {
-	background-image: url("chevron_up.svg");
-}
-.vs-dark .codicon.arrow-up,
-.hc-black .codicon.arrow-up {
-	background-image: url("chevron_up_inverse.svg");
-}
 
-.codicon.arrow-down {
-	background-image: url("chevron_down.svg");
-}
-.vs-dark .codicon.arrow-down,
-.hc-black .codicon.arrow-down {
-	background-image: url("chevron_down_inverse.svg");
+
+/* Icons as masked elements for easy theme switching.
+Includes non-masked style declarations. */
+.masked {
```

Comment:
So what's the benefit of doing this vs just having the dark-theme scoped selectors?

Reply:
Ok so it looks like it's for being able to have the color be theme-controlled. Why did you choose to do that for these buttons? All other icons in the app that we contribute just have dark and light versions.

If we really want to go the route of having theme-aware icons then I think we should look into hooking into what VS Code has set up with their icon font so that we stay more consistent in how icons are treated across the app.

Reply:
I did all this because the design direction departs from the existing pattern. It now indicates unique colors for the icons and other UI elements in each of the three primary color themes: light, dark and hc. If you have not done so already, I encourage you to familiarize yourself with the design work recently done: https://www.figma.com/file/QRkuP8YG7sZr87LRoS210ldo/Azure-Data-Studio
Look for **Toolbar** with the region: **Latest**.